test/Microsoft.Azure.WebJobs.Host.UnitTests/AttributeClonerTests.cs

```diff
@@ -251,6 +312,27 @@ public void Setting_WithNoValueInResolver_Throws()
         }
 
         [Fact]
+        public void AppSettingAttribute_DoesNotResolveBrackets()
```

Comment:
Prop1 won't be unchanged. It will resolve to EnvironmentVariable["test{x}and%y%"], if such a variable exists, otherwise it will throw.